app/grandchallenge/cases/models.py

```diff
@@ -948,3 +983,85 @@ def start_dicom_import_job(self):
             self._mark_failed(
                 error_message="An unexpected error occurred", exc=e
             )
+
+    def get_job_output_manifest(self, *, event):
```

Comment:
This is returning the job summary from the output manifest, so maybe rename?

```suggestion
    def get_job_summary(self, *, event):
```